### DISCUSSION TITLE
Floem Themed Widget Gallery (https://github.com/moon989/floem-scorpionui)

### DISCUSSION THREAD

**Discussion by moon989** [lapce/floem#582] [Show and tell]
https://github.com/moon989/floem-scorpionui

![image](https://github.com/user-attachments/assets/99b62a10-7554-4199-8040-73b522e8e267)
![image](https://github.com/user-attachments/assets/e0546eb4-0fc4-4acf-96e0-5f6d21e34f52)

**Comment by jrmoulton** [comment]
Ooh nice. Is there a reason that you aren't using the custom style pattern like how the slider does for customizing props?

**Reply by moon989** [reply]
> Ooh nice. Is there a reason that you aren't using the custom style pattern like how the slider does for customizing props?

which props?
